shared/SampleShared/SampleShared_win32.vcxproj.filters

```diff
@@ -39,6 +39,5 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />
-    <None Include="$(OpenXRLoaderBinaryRoot)\bin\openxr_loader.dll" />
```

Comment:
Was this intended?

Reply:
This tends to flip-flop due to the NuGet package quirky behavior. It should be okay.